modules/openapi-generator/src/main/resources/Javascript-Flowtyped/api.mustache

```diff
@@ -117,12 +117,12 @@ export const {{classname}}FetchParamCreator = function (configuration?: Configur
             }
             {{/isKeyInQuery}}
             {{/isApiKey}}
-            {{#isBasic}}
+            {{#isBasicBasic}}
```

Comment:
@wing328 note: this is not only about documentation: it fixes the fact that the below lines would be generated in case of operations with bearer or signature auth